content/en/platform/corda/4.4/enterprise/cordapps/cordapp-build-systems.md

```diff
@@ -185,6 +185,22 @@ in the CorDapp JAR
 
 Note that the `cordformation` and `cordapp` Gradle plugins can be used together.
 
+### Migrating to the latest version of Corda Gradle plugins
+
+{{< note >}}
+The latest version of Corda Gradle plugins is 5.1.x, which require Gradle 7.
+{{< /note >}}
+
+The `cordformation` plugin has been updated to resolve some confusion surrounding its use. It now creates the following Gradle configurations:
+
+* `cordapp` - Used for any CorDapps you want to deploy, excluding any CorDapp built by the local project.
+* `cordaDriver` - Used for any artifacts that must be added to each node's drivers/directory, for example database drivers or the Corda shell.
+* `corda` - The Corda artifact itself, or the Corda TestServer.
+* `cordaBootstrapper` - Used for Corda's Bootstrapper artifact, i.e. a compatible version of `corda-node-api`. You may also wish to include an implementation of SLF4J for the Bootstrapper to use, for example `slf4j-simple`.
```

Comment:
```suggestion
* `cordaBootstrapper` - Used for Corda's Bootstrapper artifact; i.e. a compatible version of `corda-node-api`. You may also wish to include an implementation of SLF4J for the Bootstrapper to use; for example, `slf4j-simple`.
```

Not sure what the house style is about i.e, etc. and so on. I hope use "that is" instead of i.e. but I would go by our internal styles.

Reply:
I'll leave as i.e. for now but will put it in the list for discussion at the next bi-weekly sync